azurerm/resource_arm_streamanalytics_job_test.go

```diff
@@ -293,78 +399,337 @@ resource "azurerm_eventhub" "test" {
   message_retention   = 1
 }
 
-# Create authorization rule for listening on the event hub
 resource "azurerm_eventhub_authorization_rule" "test" {
-    name = "acctesteventhubauth-%d"
+  name 	              = "acctesteventhubauth-%d"
```

Comment:
If we're going to align these `=`, we should align them all correctly. There are a bunch of inconsistencies below.

Reply:
thanks @rickerbh - I've reformatted it.